App/Components/ProgressBar.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+
+import * as Progress from 'react-native-progress';
+
+const styles = StyleSheet.create({
+    textContainer: {
+        flexDirection: 'row',
+        justifyContent: 'space-between'
+    },
+    progressBar: {
+        backgroundColor: 'rgb(239, 240, 241)',
+        borderColor: 'rgb(239, 240, 241)'
+    }
+});
```

Comment:
Can you create a file ProgressBarStyles.js, put this in that file, and then import styles from that folder.